cfgov/agreements/jinja2/agreements/_database_disclaimer.html

```diff
@@ -1,14 +1,72 @@
 {% import "molecules/social-media.html" as social_media with context %}
 
-<div class="block">
-    <hr>
-</div>
+<div class="block block__padded-top block__border-top">
+    <h2 id="find-issuer">
+        Can't find an issuer?
+    </h2>
+    <p>
+        The drop-down list contains credit card issuers that have
+        submitted credit card agreements as required under
+        Section 204 of the Credit Card Accountability
+        Responsibility and Disclosure Act of 2009 (the CARD Act).
+        If you cannot find a card issuer, it may be that:
+    </p>
+    <ul>
+        <li>
+        The issuer has not yet submitted its agreements
+        to the Federal Reserve Board;
+        </li>
+        <li>
+        The issuer has fewer than 10,000 accounts and does not
+        have to submit its agreements; or
+        </li>
+        <li>
+        The name that you are looking for is not the actual
+        issuer's name
+        (for example, "ABC Card" is issued by XYZ Bank).
+        </li>
+    </ul>
+    <p>
+        If none of these reasons apply and you still cannot find
+        an agreement, call them to request a copy of your agreement.
+        Under federal law, your credit card issuer is required to
+        provide a copy of your agreement upon request.
+    </p>
+    <p>
+        Look on the back of the credit card or on your latest
+        monthly statement to find the name of the issuer. The bank,
+        retailer, or other entity whose name appears on the front
+        of the card may not be the actual issuer of the card.
+        You can also check the issuer's website; typically,
+        the bank name is listed at the bottom of the issuer's
+        homepage.
+    </p>
+    <p>
+        <a href="#">
```

Comment:
Should this link anywhere?

Reply:
Fixed in 3a92a97. I originally though jumping back to the top of the page made sense, but I've reconsidered and added an anchor right to the header above the issuer select on each page.